run_ci.sh

```diff
@@ -104,7 +104,8 @@ if [ -d "/home/notstack/ironic-image" ] ; then
     export IRONIC_LOCAL_IMAGE=https://github.com/metal3-io/ironic-image
     export UPSTREAM_IRONIC=true
 fi
-if [ -d "/home/notstack/ironic-inspector-image" ] ; then
+# Starting from Openshift 4.9 the ironic-inspector container is not used anymore
+if [ "$OPENSHIFT_VERSION" <= "4.8" ] && [ -d "/home/notstack/ironic-inspector-image" ] ; then
```

Comment:
Similarly this isn't a valid test, I think you'll see it generates a syntax error?